apps/docusaurus/docs/integration/wallet-adapter-concept.md

```diff
@@ -26,8 +23,8 @@ Implementing wallet integration can be difficult for dapps in:
 In addition, creating and implementing a wallet is also not an easy task,
 
 1. Provide a wallet that follows a known standard, so it is easy to integrate with
-2. Getting visibility and exposure in the ecosystem among all the other wallets
-3. Dapp projects need to dedicate time and resource to integrate the wallet within their app
+2. Dapp projects need to dedicate time and resource to integrate the wallet within their app
+3. Wallets require to rely solely on a dapp integration
```

Comment:
I'm not sure what this means?

Reply:
yea... basically it means that wallets have to rely on dapp integration effort while with the adapter wallets are already provided

Reply:
Ok it might be clearer to write something like 

```suggestion
3. dApps need to spend extra integration effort to incorporate each wallet with the legacy wallet adapter
```